rudder-server-relay/SPECS/rudder-server-relay.spec

```diff
@@ -318,10 +318,17 @@ fi
 
 %if 0%{?suse_version}
 # Add required includes in the apache2 configuration
-if ! grep -qE "^. /etc/sysconfig/rudder-relay-apache$" /etc/sysconfig/apache2; then
+nextline=$(grep -A1 -E "^. /etc/sysconfig/rudder-relay-apache$" /etc/sysconfig/apache2 | tail -n1)
+if [ "${nextline}" = "" ]; then
+  # No include currently
   echo -e '# This sources the modules/defines needed by Rudder\n. /etc/sysconfig/rudder-relay-apache' >> /etc/sysconfig/apache2
```

Comment:
Why don' we do 2 lines instead of one for each echo ? this makes it completely unreadable

Reply:
echo -e '# This sources the modules/defines needed by Rudder.' >> /etc/sysconfig/apache2 /etc/sysconfig/rudder-relay-apache'  >> /etc/sysconfig/apache2